archeryutils/targets.py

```diff
@@ -84,28 +114,31 @@ class Target:
 
     """
 
+    _face_spec: FaceSpec
+    supported_systems = get_args(ScoringSystem)
+    supported_distance_units = Length.yard | Length.metre
+    supported_diameter_units = Length.cm | Length.inch | Length.metre
```

Comment:
Also considered `mm` for beiter hit miss discs as thats the unit I'd refer to them in, but out of scope for this I think? And trivial to add in a seperate pr after this since the supported units are now defined in a single place.

Reply:
Added comment https://github.com/jatkinson1000/archeryutils/pull/36#issuecomment-1982596587 to log this.